dotcom-rendering/src/types/content.ts

```diff
@@ -2,6 +2,8 @@ import type { ArticleTheme } from '@guardian/libs';
 
 export type StarRating = 0 | 1 | 2 | 3 | 4 | 5;
 
+export type BoostLevel = 'default' | 'boost' | 'megaboost' | 'gigaboost';
```

Comment:
Given that boost level is optional, I wonder if we want/need to have 'default'? My thought is that a level should reflect a type of boosting i.e. 'boost', 'megaboost', 'gigaboost' and that if a card has not been boosted then it shouldn't have a boost level, rather than return "default".

Reply:
So this is effectively just replicating the model defined in [facia scala client here](https://github.com/guardian/facia-scala-client/blob/main/fapi-client/src/main/scala/com/gu/facia/api/utils/ResolvedMetaData.scala#L11) (and probably further back in the fronts tool somewhere). 

Boostlevel is being set as optional here because, from the testing I did, my understanding is that we'll soon end up with fronts in two states: those recently launched and pressed will have a boostLevel in their display settings (alongside isBoosted, etc.), while fronts that have been launched before the addition of boostLevel won't - and that's the case we need to handle for now. 

This seems to work, but I can imagine there's better alternatives to this!